src/planscape/conditions/views.py

```diff
@@ -34,7 +37,29 @@ def get_wms(params: QueryDict):
         row = cursor.fetchone()
     return row
 
+def get_region(params: QueryDict):
+    # Get region name
+    assert isinstance(params['region_name'], str)
+    region_name = params['region_name']
+    print(region_name)
+
+    # Read from conditions config
+    config_path = os.path.join(
+       PLANSCAPE_ROOT_DIRECTORY, 'Planscape/src/planscape/config/conditions.json')
+    conditions_config = json.load(open(config_path, 'r'))
+
+    # Extract specific region data from JSON
+    for region in conditions_config['regions']:
+        print(region)
```

Comment:
remove

Reply:
did you mean for me to just remove the debug print statements?

Reply:
yup!